openpype/hosts/houdini/plugins/publish/extract_fbx.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""Fbx Extractor for houdini. """
+
+import os
+import pyblish.api
+from openpype.pipeline import publish
+from openpype.hosts.houdini.api.lib import render_rop
+
+import hou
+
+
+class ExtractFBX(publish.Extractor):
+
+    label = "Extract FBX"
+    families = ["fbx"]
+    hosts = ["houdini"]
+
+    order = pyblish.api.ExtractorOrder + 0.1
+
+    def process(self, instance):
+
+        # get rop node
+        ropnode = hou.node(instance.data.get("instance_node"))
+
+        # render rop
+        render_rop(ropnode)
+
+        # get required data
+        file_name, staging_dir = self.get_paths_data(ropnode)
+        representation = self.get_representation(instance,
+                                                 file_name,
+                                                 staging_dir)
+
+        # set value type for 'representations' key to list
+        if "representations" not in instance.data:
+            instance.data["representations"] = []
+
+        # update instance data
+        instance.data["stagingDir"] = staging_dir
+        instance.data["representations"].append(representation)
+
+    def get_paths_data(self, ropnode):
+        # Get the filename from the filename parameter
+        output = ropnode.evalParm("sopoutput")
+
+        staging_dir = os.path.normpath(os.path.dirname(output))
+
+        file_name = os.path.basename(output)
+
+        self.log.info("Writing FBX '%s' to '%s'" % (file_name,
+                                                    staging_dir))
```

Comment:
This feels very weird as a log inside `get_paths_data`. This should really be just BEFORE the writing of the rop node.

Also note that this should be a debug log. The artist doesn't care what temp staging directory/file gets extracted (especially because the logs are not visible during publishing but only after the publisher) they just want the final publish result.